src/script/descriptor.h

```diff
@@ -32,6 +32,9 @@ struct Descriptor {
     /** Whether the expansion of this descriptor depends on the position. */
     virtual bool IsRange() const = 0;
 
+    /** Whether this descriptor has all information about signing (ignoring private keys). */
```

Comment:
Would it be useful to mention that raw and address descriptors are always considered unsolvable? It seems like this would make the description more concrete, and make it clear that a raw P2PK descriptor is not considered solvable (for example).